ods_ci/tests/Tests/100__deploy/130__operators/130__rhods_operator/144_dsc_negative_no_dependant_op.robot

```
@@ -89,9 +157,27 @@ Remove DSC And DSCI Resources
 
 Check DataScienceCluster Conditions When Dependant Operators Are Not Installed
     [Documentation]   Keyword to check the DSC conditions when dependant (servicemesh, serverless) operators are not installed
-    Log To Console    message=Checking DataScienceCluster conditions
     ${return_code}    ${output}=    Run And Return Rc And Output
     ...    oc get DataScienceCluster ${DSC_NAME} -n ${OPERATOR_NS} -o json | jq -r '.status.conditions | map(.message) | join(",")'    #robocop:disable
     Should Be Equal As Integers  ${return_code}   0   msg=Error retrieved DSC conditions
     Should Contain    ${output}    operator servicemeshoperator not found. Please install the operator before enabling kserve component    #robocop:disable
     Should Contain    ${output}    operator serverless-operator not found. Please install the operator before enabling kserve component    #robocop:disable
+
+Check DSCInitialization Conditions When Service Mesh Operator Is Not Installed
```

`DSCIInitialization Should Fail Because ServiceMesh Is Missing` maybe